Component Plug-in (CP) Examples/README.md

```diff
@@ -8,12 +8,19 @@
 
 ## Sample interface
 ```
-mapField(
-    label: "Map",
-    labelPosition: "ABOVE",
-    validations: {},
-    height: "AUTO",
-    key: <YOUR_API_KEY>,
-    location: "America"
+load(
+  local!pins,
+  {
+    a!textField(value: local!pins, label: "Pins", saveInto: local!pins),
+    mapField(
+      label: "Map",
+      labelPosition: "ABOVE",
+      validations: {},
+      height: "AUTO",
+      key: <YOUR_API_KEY>,
```

Comment:
just a thought, would it be clearer if we renamed this to `apiKey` instead?

Reply:
Slightly, although that's not the component I'm modifying in this PR (was just updating the README bc I couldn't help myself)